profile.d/stable

```diff
@@ -2,7 +2,7 @@
 # WARNING: Any variables defined here, override those defined from the Jenkinsfile
 #
 #
-RELEASE_GIT_BRANCH=master
+RELEASE_GIT_BRANCH=stable-2.414
```

Comment:
Since this is in the `stable` profile, this makes sense to me, though I'm not sure why the change was not already made when the stable-2.414 branch was created.  I'll need to study further to answer my questions
```suggestion
RELEASE_GIT_BRANCH=stable-2.414
```